tests/Fixtures/Integration/misc/PHP7.test

```diff
@@ -25,7 +25,6 @@ use const some\x\E;
 use function some\a\fn_a;
 use function some\a\fn_b;
 use function some\a\fn_c;
-use function some\x\A\B;
```

Comment:
?

Reply:
This import is not used in the input code, but this was not detected by the fixer before.